cumulus_library/studies/core/medication_request.sql

```diff
@@ -8,14 +8,14 @@ SELECT
     date_trunc('month', date(from_iso8601_timestamp(mr.authoredon)))
     AS authoredon_month,
     mr.category,
-    med_coding.code AS code,
-    med_coding.display AS display,
-    med_coding.system AS system, --noqa: RF04
+    cm.code AS medication_code,
+    cm.display AS medication_display,
+    cm.code_system AS medication_system,
```

Comment:
@comorbidity I'm labelling these because i am assuming, at some point, we're also going to want to unnest mr.category, which is also a codeable concept, so we'll need to keep these distinct from each other

Reply:
elsewhere these fields are called 
rx_code
rx_display
rx_system

Reply:
alternatively 
med_code 
med_display 
med_system